label.py

```diff
@@ -375,7 +380,7 @@ def images_cmp(x, y):
 def group_images():
     global images_to_show
     sorted_images_to_show = sorted(images_to_show, key=functools.cmp_to_key(images_cmp))
-    bug_ids = set([file_name.split('_')[0] for file_name in sorted_images_to_show])
+    bug_ids = OrderedDict.fromkeys([file_name.split('_')[0] for file_name in unlabelled + unlabelled_in_all_labels])
```

Comment:
We want to show the `unlabelled_in_all_labels` first

Reply:
It was a naming confusion. `unlabelled_in_all_labels` actually meant `unlabelled_but_present_in_all_labels`

Reply:
Ahahah, yes, thanks!